source/simulator/assembly.cc

```diff
@@ -457,9 +454,7 @@ namespace aspect
 #ifdef ASPECT_USE_PETSC
     Amg_data.symmetric_operator = false;
 #else
-#if DEAL_II_VERSION_GTE(9,2,0)
     Amg_data.constant_modes = constant_modes;
-#else
```

Comment:
You need to remove this whole code block up unto the #endif. Maybe you did that and it got lost in our rebase this morning. But now you can simply add this change and do git push like normal.